schema.graphql

```diff
@@ -189,26 +200,26 @@ type PlanTemplate @entity {
 type Plan @entity {
   id: ID! # Index, hex encoded
   price: BigInt!
-  creator: String! # Address
+  creator: String! @index(unique: false) # Address
   planTemplate: PlanTemplate!
   active: Boolean!
 
-  deployment: Deployment
+  deployment: Deployment 
 
   createdBlock: Int
   lastEvent: String #topicHandler:blockNumber
 }
 
 type ServiceAgreement @entity {
   id: ID! # contract address
-  indexerAddress: String!
-  consumerAddress: String!
-  deployment: Deployment!
+  indexerAddress: String! @index(unique: false)
```

Comment:
Can use `@index` directly as `unique: false` should be by default, correct me if that's not true